front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart/drawTrain.jsx

```diff
@@ -102,11 +102,7 @@ export default function drawTrain(
           `${isSelected && 'selected'} route-aspect`,
           ecoRouteAspect,
           groupID,
-          'curveLinear',
```

Comment:
all of this was unecessary ?

Reply:
Yes, it was not used in the function (or could be replaced by a constant) :+1: